zio-http/src/main/scala/zhttp/api/API.scala

```diff
@@ -0,0 +1,87 @@
+package zhttp.api
+
+import zhttp.http.{Headers => _, Path => _}
+import zio.schema.Schema
+
+/**
+ *   - Input and Output as Schemas.
+ *   - Support wide range of Codecs including Json
+ *     - Dynamically decide response format based upon Request Header
+ */
+final case class API[Params, Input, Output](
+  method: HttpMethod,
```

Comment:
```suggestion
  method: Method,
```

Reply:
One potential reason not to do that is so we can break the model into a _shared_ project so that the API DSL can be interpreted into Scala.js fetch requests. Though perhaps the `Method` sealed trait could be moved there as well, so long as it isn't part of `netty`.